### PR TITLE
Closing price tag implemented

### PR DESCRIPTION
**Pull request by djmiley**
Get rid of open/close labels/ lines now?

### CODE REVIEW

**Comment by jleft** [comment]
Look really good, nice work :-)

> Get rid of open/close labels/ lines now?

Yeah, I think that'd be a good idea :+1:

File: src/assets/js/main.js [hunk 1]
```diff
@@ -88,16 +88,52 @@
...
+
+    var priceFormat = d3.format('.2f');
+
     var startPriceLine = fc.annotation.line()
         .orient('horizontal')
         .value(function(d) { return d.open; })
         .label(function(d) { return 'OPEN'; });
 
-    var endPriceLine = fc.annotation.line()
+    var closePriceLine = fc.annotation.line()
```
Comment (jleft) [inline src/assets/js/main.js]: I can't see much value in both `closePriceLine` and `closeAxisAnnotation`, so I think we're good to remove `closePriceLine`?

**Comment by djmiley** [comment]
`openPriceLine` feels a bit redundant to me, especially with respect to moving the data back further in time?

**Comment by djmiley** [comment]
Keep the drawn close price dotted line?

**Comment by jleft** [comment]
> openPriceLine feels a bit redundant to me, especially with respect to moving the data back further in time?

That's been mentioned before, so let's go with the consensus and remove it.

> Keep the drawn close price dotted line?

Yes, I think so.

**Comment by jleft** [comment]
I expect, as `linearTimeSeries` plots the axis over the plot area, that the axis labels will overlap, above, the annotation's text?

**Comment by jleft** [comment]
Resolves #69

**Pull request merged**